crates/lang/macro/tests/ui/pass/07-flipper-as-dependency.rs

```diff
@@ -2,14 +2,26 @@ use ink_lang as ink;
 
 #[ink::contract(compile_as_dependency = true)]
 mod flipper {
+    #[ink_lang::trait_definition]
+    pub trait FlipperTrait {
+        #[ink(constructor)]
+        fn new() -> Self;
+
+        #[ink(message)]
+        fn flip(&mut self);
+
+        #[ink(message)]
+        fn get(&self) -> bool;
+    }
+
     #[ink(storage)]
     pub struct Flipper {
         value: bool,
     }
 
     impl Flipper {
         #[ink(constructor)]
-        pub fn new(init_value: bool) -> Self {
+        pub fn new2(init_value: bool) -> Self {
```

Comment:
Instead of doing this I would add a new UI test instead. This way we have the bug isolated